### ISSUE TITLE
Confused by GO:0061990, molecular_function "beta-ketodecanoyl-[acyl-carrier-protein] synthase"

### ISSUE THREAD

**Issue by baoilleach** [geneontology/helpdesk#103]
For the most part, molecular functions appear to avoid colliding with names of proteins, sometimes by appending the term "activity". However, there are a number of terms whose names are synthases rather than "synthase activity". For example...

GO:0061990 has the name beta-ketodecanoyl-[acyl-carrier-protein] synthase

But Uniprot Q9HYV7 (http://www.uniprot.org/uniprot/Q9HYV7) is named "Beta-ketodecanoyl-[acyl-carrier-protein] synthase". It even is annotated with GO terms, none of which is GO:0061990. It also has a Wikipedia page (https://en.wikipedia.org/wiki/Beta-ketodecanoyl-(acyl-carrier-protein)_synthase).

In short, is this an error in GO, or have I made an error?

**Comment by kltm** [comment]
@ukemi @vanaukenk 

Would either of you know the right person to contact internally to answer this question?

**Comment by suzialeksander** [comment]
@sandyl27 , has this issue been resolved? I see the term name for GO:0061990 has been fixed, but I don't see any annotations to this term. Can you review the protein mentioned?
Thanks

**Comment by sandyl27** [comment]
I have looked at the annotations in CACAO and have fixed them.
